Migration-5-6.md

```diff
@@ -13,7 +14,7 @@ Changes in Scala API:
 See [related section](README.md#providing-converters-for-producer/consumer) in docs.
 1. The `Delivery` is now sealed trait - there are `Delivery.Ok[A]` (e.g. `Delivery[Bytes]`, depends on type-conversion) and `Delivery.MalformedContent`.
 After getting the `Delivery[A]` you should pattern-match it.
-1. The API now requires an implicit `monix.execution.Scheduler` instead of `ExecutionContext`.
+1. The API of 6.0.x now requires an implicit `monix.execution.Scheduler` instead of `ExecutionContext` **(Changed back in 6.1.x)**
```

Comment:
In general, I would prefer to have these migration instruction on [Releases page](https://github.com/avast/rabbitmq-scala-client/releases).

Reply:
Good point. Will be added in time of new release :-)